docs/advanced/pycpp/object.rst

```diff
@@ -33,19 +33,91 @@ The reverse direction uses the following syntax:
 
 When conversion fails, both directions throw the exception :class:`cast_error`.
 
+.. _python_libs:
+
+Accessing Python libraries from C++
+===================================
+
+It is also possible to import types and classes defined in the Python standard
```

Comment:
A type and a class are the same thing in Python. Maybe replace "types and classes" with "types and functions".

Reply:
I went with objects since that should cover everything.